udlg/structure/structure.py

```diff
@@ -17,11 +17,12 @@
 from .constants import (
     BYTE_SIZE, INT_SIZE, RecordTypeEnum
 )
-from .base import SimpleSerializerMixin
+from .base import SimpleSerializerMixin, BinaryRecordStructure as brs
```

Comment:
Seems we don't need this import as we don't use it. Moreover please avoid shortcuts as far as it would be hard to understand anything soon enough as they count grow